juju/juju
# Attach a cookie jar from the client store

The `connector.ClientStoreConnector` implementation of `connector.Connector` should be able to use the macaroon stored in the local share (client store).  This change plugs the client store cookie jar into the dial options bakery client so that the saved macaroon is used if available.

## QA steps

This changes means that a user that hasn't got their password stored in the local share can still log in as long as there is a valid macaroon in the cookie store.

These steps ensure the above holds:

```sh
juju logout
juju login -u <user>
```

Then it is possible to obtain an authenticated connection using `apiconnector.New(...)`
